bpo-44747: Refactor usage of sys._getframe at typing module (#27387)
python/cpython: Misc/NEWS.d/next/Library/2021-07-27-12-06-19.bpo-44747.epUzZz.rst

> Reduce usage of ``sys._getframe`` at ``typing`` module. Patch provided by

Reviewer: ```suggestion
Refactor usage of ``sys._getframe`` in ``typing`` module. Patch provided by
```
